jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -71,6 +71,35 @@
     utf8_ptr ## must_be_freed = true; \
   }
 
+#ifdef ECMA_VALUE_CAN_STORE_UINTPTR_VALUE_DIRECTLY
+
+/**
+ * Set an internal property value of pointer
+ */
+#define ECMA_SET_INTERNAL_VALUE_POINTER(field, pointer) field = (ecma_value_t) pointer;
+
+/**
+ * Get an internal property value of pointer
+ */
+#define ECMA_GET_INTERNAL_VALUE_POINTER(type, field) (type *) field;
```

Comment:
Add a parenthesis around the expression.
```C++
#define ECMA_GET_INTERNAL_VALUE_POINTER(type, field) \
((type *) field)
```

Note: Don't close these with semicolons.